build/resources/main/knowledge_environment.graphqls

```diff
@@ -2,6 +2,7 @@ type Query {
   genes(symbol: String): [MyGeneInfoHit]

```

Comment:
I don't think we need to check this file in. Not going to hold back the card, but we should revisit our .gitignore to remove the files under the build directory.